Dockerfile

```diff
@@ -27,6 +29,7 @@ RUN yarn run build
 
 FROM base as production
 ENV NODE_ENV=production
+RUN rm -rf ./nitro-web/components ./nitro-web/store
```

Comment:
@mattwr18 unfortunately, this adds another layer to the `production` stage and therefore it won't reduce the image size.

Reply:
Copy all into builder and then pick what you need out if that image by specifying the image (see backend build)

Reply:
> Why not just copy all into the build image and trusting the dockerignore file? @mattwr18

When you say copy all into the build image, what do you mean by that? if you look at this line https://github.com/Human-Connection/Nitro-Web/blob/9f8a292c7e39030a4fb05622c8830e5dbff967db/Dockerfile#L27 we are copying all; however, this all is only what has been copied from the host into the base image.

Reply:
> @mattwr18 unfortunately, this adds another layer to the `production` stage and therefore it won't reduce the image size.

I would have thought so, as it does add another layer, although, we checked the image size both before and after adding this line and did not observe a difference.

Reply:
Just as a note from my experience, you can’t copy stuff that’s ignored, even if you reference it one by one. Maybe that helps, was a catch for me.

Reply:
That is my understanding as well; however, it's not clear to me that everything that should be in the dockerignore file actually is.